rafter/step1.md

```diff
@@ -1,22 +1,7 @@
-WHAT ARE THE PRE_STEPS A USER CAN SEE IN THE TERMINAL? IS IT HELM INSTALLATION / TILLER INSTALLATION / MINIKUBE SET-UP? 
-Make sure the environment is set up. Check if the API Server is running. 
-It make take at least 1 minute, Minikube needs time, be patient.
-`kubectl get pod -n kube-system | grep kube-apiserver-minikube`{{execute}} - DOES THE COMMAND CHECK IF MINKUBE IS SET UP CORRECTLY?
+The environment is set up and ready to start palying with Rafter.
```

Comment:
rather `Wait for the environment to finish setting up and then start playing with Rafter.`